test/Extensions.Mcp.E2ETests/Fixtures/McpEndToEndProjectFixtures.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using static Microsoft.Azure.Functions.Extensions.Mcp.E2ETests.Fixtures.McpEndToEndProjects;
+
+namespace Extensions.Mcp.E2ETests.Fixtures;
+
+public class McpEndToEndProjectFixtures
+{
+    // These fixtures should be used only as class fixtures. Setting them as collection fixtures creates coupling across test classes.
+
+    public class InProcProjectFixture() : McpEndToEndFixtureBase(new InProcNet8Project("TestApp")) { }
+
+    public class DefaultProjectFixture() : McpEndToEndFixtureBase(new DotnetWorkerProject("TestAppIsolated")) { }
```

Comment:
You wouldn't be able to test the isolated ext using `Microsoft.Azure.Functions.Extensions.Mcp` - are we lumping all the E2E tests in one place? I think we should have an E2E test project for each one (`Microsoft.Azure.Functions.Extensions.Mcp` and `Microsoft.Azure.Functions.Worker.Extensions.Mcp`)